<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music_Songs_Add</class>
 <widget class="QDialog" name="Music_Songs_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Song</string>
  </property>
  <widget class="QLabel" name="AddSong_MainText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>19</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add New Song</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddSong_CommentsBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_TitleText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song Title</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="AddSong_DateBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_DateText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Release Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_NumberText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_KeyWordsText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Words</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddSong_TitleBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_CommentsText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddSong_KeyWordsBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="AddSong_NumberBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddSong_SaveButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_CodeText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Authorisation Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddSong_CodeBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddSong_CancelButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_TitleValidation">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_KeyValidation">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_CommentsValidation">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddSong_CodeValidation">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AddSong_SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Add</receiver>
   <slot>AddSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddSong_CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Add</receiver>
   <slot>CancelButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>688</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddSong()</slot>
  <slot>CancelButton()</slot>
 </slots>
</ui>
